patches/active-model-adapter.patch

```diff
@@ -0,0 +1,67 @@
+diff --git a/addon/active-model-adapter.ts b/addon/active-model-adapter.ts
+index 8d90a844c16ca465aad5113bbfc6e58f98b5124e..017869863a882a46e3449209654c81ca65ef9f55 100644
+--- a/addon/active-model-adapter.ts
++++ b/addon/active-model-adapter.ts
+@@ -1,8 +1,5 @@
+ import RESTAdapter from '@ember-data/adapter/rest';
+-import AdapterError, {
+-  InvalidError,
+-  errorsHashToArray,
+-} from '@ember-data/adapter/error';
++import AdapterError, { InvalidError } from '@ember-data/adapter/error';
+ import { pluralize } from 'ember-inflector';
+ import { AnyObject } from 'active-model-adapter';
+ import { decamelize, underscore } from '@ember/string';
+@@ -162,3 +159,52 @@ export default class ActiveModelAdapter extends RESTAdapter {
+     }
+   }
+ }
++
++/**
++ * Adding errorsHashToArray() func until next version of active-model-adapter is released, this function is removed in
++ * ember-data 5+
++ *
++ * https://github.com/adopted-ember-addons/active-model-adapter/issues/191
++ * https://github.com/adopted-ember-addons/active-model-adapter/pull/192
++ */
```

Comment:
blerg

ember-data removed a function that active-model-adapter relies on. There's a PR to add it back, which was merged, but a new version of that addon has not been published yet.

Patching it in for now. This will only affect our test-app, to make sure the tests pass for the ED 5 + ActiveModel scenarios. Users on ED 5 + ActiveModel will have to do the same (regardless of whether they use factory guy addon or not) so we can assume they have done so.